src/components/KudosRequest.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react'
+import Request, { RequestMaker } from './Request'
+
+export default class KudosRequest extends Component {
+    static propTypes = RequestMaker.handlersPropTypes
+    static defaultProps = RequestMaker.defaultProps
+
+    normalize = kudos =>
+        kudos.reverse()
```

Comment:
oh, you left this method, is it really alright?

Reply:
What do you mean? This behaviour is copied from its initial place in the Wall component. No business logic changed, AFAIK.